src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -77,6 +77,8 @@ public static partial class PlatformDetection
 
         public static bool IsUsingLimitedCultures => !IsNotMobile;
         public static bool IsNotUsingLimitedCultures => IsNotMobile;
+        public static bool IsNotRunningOnMacOS => !RuntimeInformation.RuntimeIdentifier.StartsWith("iossimulator")
```

Comment:
According to this name it should also include `!IsOSX`

Reply:
Removed the parameter, extanded IsCaseSensitiveOS parameter.